src/app/components/shared/cms/cms.component.ts

```diff
@@ -30,7 +39,12 @@ export class CmsComponent extends WithUnsubscribe() implements OnInit {
 
     this.cms.get(this.page).subscribe(
       (blob) => {
-        this.blob = blob;
+        // Using html fragments instead of angular sanitization sanitization
+        // uses innerHTML to insert html blobs, and this will not run <script>
+        // tags. This is a safety feature of innerHTML and cannot be bypassed.
```

Comment:
```suggestion
        // Using html fragments instead of innerHTML. 
        // In the HTML5 spec, script tags that are inserted via InnerHTML will not be executed.
        // Using a document fragment allows us to insert any tag.
        // NOTE: Angular's Sanitization is ignored since we are bypassing Angular's normal rendering system.
        // NOTE: It might be useful to consider using ShadowDom to isolate these CMS HTML fragments from 
        //       the rest of the site. This would prevent, for example, a careless CSS global style in the CMS fragment
        //       from affecting the rest of the angular site.
```